src/kirby/scss/web/_web-styles.scss

```
@@ -36,3 +36,12 @@
         align-items: flex-end;
     }  
 }
+
+.alert-btn {
```

Be consistent with modal + action-sheet:
```suggestion
.kirby-alert-btn {
```